clang/lib/Basic/Targets/Mips.h

```diff
@@ -330,6 +331,8 @@ class LLVM_LIBRARY_VISIBILITY MipsTargetInfo : public TargetInfo {
         IsMicromips = true;
       else if (Feature == "+mips32r6" || Feature == "+mips64r6")
         HasUnalignedAccess = true;
+      else if (Feature == "+strict-align")
+        StrictAlign = true;
```

Comment:
Why not clear HasUnalignedAccess directly here? That's what AArch64 and ARM do.

Reply:
We cannot due to that +strict-align may be listed before +mips32/64.

I have some try. In all my try, +mips32/64 appeared before +strict-align.
But I am not sure that +mips32/64 will always appear before +strict-align.

Reply:
Oh, I see, that makes sense, but a comment would be helpful -- and testcases verifying either ordering still DTRT.

Reply:
> Oh, I see, that makes sense, but a comment would be helpful -- and testcases verifying either ordering still DTRT.

I will add a comment. But in fact I cannot figure out a case that +strict-align appears before +mips32r6/+mips64r6.